
float32 amb_vel_1
float32 amb_vel_2
---
float32 reward
